src/gallery/slider.js

```diff
@@ -79,12 +79,14 @@ const clientWidth = window.innerWidth,
 			},
 
 			author = mediaInfo.author ? mediaInfo.author : msg( lang, 'gallery-unknown-author' ),
-			link = mediaInfo.filePage
+			link = mediaInfo.filePage,
+			description = getImageDescription()
 
 		// @todo consider a wrapper container for all the image info?
```

Comment:
Did consider a wrapper but still not convinced, we could revisit in [T262373](https://phabricator.wikimedia.org/T262373)